build

```diff
@@ -16,9 +16,16 @@ eval $(go env)
 if [ ${GOOS} = "linux" ]; then
 	echo "Building fleetd..."
 	CGO_ENABLED=0 go build -o bin/fleetd -a -installsuffix netgo -ldflags '-s' ${REPO_PATH}/fleetd
```

Comment:
Because test script remove "-e", here need to check the result for each build. Otherwise, the test script will go ahead even if build failing.